components/setting_item_max.jsx

```diff
@@ -72,13 +73,11 @@ export default class SettingItemMax extends React.Component {
         var submit = '';
         if (this.props.submit) {
             submit = (
-                <input
-                    id='saveSetting'
-                    type='submit'
-                    className='btn btn-sm btn-primary'
-                    href='#'
+                <SaveButton
+                    saving={this.props.saving}
+                    disabled={this.props.saving}
                     onClick={this.props.submit}
-                    value={Utils.localizeMessage('setting_item_max.save', 'Save')}
+                    messageId='setting_item_max.saving'
```

Comment:
I think this should be `savingMessgeId`?

Reply:
Indeed!